doc/sphinx/Clusters_from_Scratch/fencing.rst

```diff
@@ -124,17 +124,17 @@ For this example, assume we have a chassis containing four nodes
 and a separately powered IPMI device active on 10.0.0.1. Following the steps
 above would go something like this:
 
-Step 1: Install the **fence-virt** package on both nodes.
+Step 1: Install the **fence-agents-ipmilan** package on both nodes.
 
 Step 2: Configure the IP address, authentication credentials, etc. in the IPMI device itself.
 
-Step 3: Choose the **fence_virt** STONITH agent.
+Step 3: Choose the **fence_ipmilan** STONITH agent.
 
 Step 4: Obtain the agent's possible parameters:
 
 .. code-block:: none
 
-    [root@pcmk-1 ~]# pcs stonith describe fence_virt
+    [root@pcmk-1 ~]# pcs stonith describe fence_ipmilan
```

Comment:
You can actually install fence_ipmilan and run pcs stonith describe for it on any host, to get the correct output to use here. You can install as many fence agents as you want without affecting the cluster, it's only when creating a fence device that the agent actually can be used by the cluster.